libyul/backends/evm/NoOutputAssembly.cpp

```diff
@@ -149,14 +149,14 @@ NoOutputEVMDialect::NoOutputEVMDialect(shared_ptr<EVMDialect> const& _copyFrom):
 	{
 		size_t parameters = fun.second.parameters.size();
 		size_t returns = fun.second.returns.size();
-		fun.second.generateCode = [=](FunctionCall const&, AbstractAssembly& _asm, std::function<void()> _visitArguments)
+		fun.second.generateCode = [=](FunctionCall const&, AbstractAssembly& _assembly, std::function<void()> _visitArguments)
```

Comment:
Is `_asm` a keyword in msvc?

Reply:
Yes, I think I stumbled across that multiple times already. Might warrant a style check.